sourcetools/com.ibm.uma/com/ibm/j9/uma/configuration/ConfigurationImpl.java

```diff
@@ -67,96 +67,98 @@
 import freemarker.template.TemplateModelIterator;
 
 public class ConfigurationImpl implements IConfiguration, ISinglePredicateEvaluator {
-		
-	String[] phaseList; 
+
+	String[] phaseList;
 	String[] excludeArtifacts;
-	
+
 	public int numberOfPhases() {
 		return phaseList.length;
 	}
-	
+
 	public String phaseName(int phase) {
-		if (phase >= numberOfPhases() )	return "base phase identifier [" + phase + "]";
+		if (phase >= numberOfPhases())
```

Comment:
If you're going to split onto 2 lines, then `{` `}` should be used.

Reply:
Sure, I'll fix that.